chapter1/configuration.md

```diff
@@ -1,4 +1,55 @@
-- Webportal login
-- navigators, options
-  (Screen shots)
+## Basic Configuration
+
+### Launch webportal
+In `appliance_console`, we can check either `ManageIQ Server` is running or not. If not then we can start it by entering digit `14` in the advanced setting.  After that when we hit IPv4 address which is given in `appliance_console` in web browser we can see the ManageIQ's Log In page.
```

Comment:
As log in is noun here, it should be written as `login` 
https://writingexplained.org/log-in-or-login-difference